code/messaging/src/test/java/com/adobe/marketing/mobile/MessagingPushPayloadTests.java

```diff
@@ -105,6 +103,7 @@ private Map<String, String> getMockData() {
         mockData.put(MessagingConstant.PushNotificationPayload.ACTION_TYPE, mockActionType);
         mockData.put(MessagingConstant.PushNotificationPayload.ACTION_URI, mockActionUri);
         mockData.put(MessagingConstant.PushNotificationPayload.ACTION_BUTTONS, mockActionButtons);
+
```

Comment:
nit: May be can we add one more tests with malformed Action button json and verify they don't crash